### DISCUSSION TITLE
Why do you choose the learn React?

### DISCUSSION THREAD

**Discussion by NobleOsinachi** [NobleOsinachi/vidly-react#8] [Q&A] [answered]
There are many other choices, but why React?

**Comment by iamGolden** [comment, marked as answer]
High demand in the job market: React is a popular JavaScript library used to build user interfaces, and there is a high demand for developers with React skills in the job market.

Large community support: React has a large and active community of developers who contribute to its development, create libraries, and share knowledge through online forums and documentation. This community support makes it easier for developers to learn and use React.

Component-based architecture: React's component-based architecture makes it easy to build complex user interfaces by breaking them down into smaller, reusable components. This approach promotes code reusability and maintainability.

Performance: React uses a virtual DOM to optimize the rendering of components, resulting in faster performance compared to traditional DOM manipulation.

Cross-platform compatibility: React can be used to build web applications as well as mobile applications using React Native, which allows for cross-platform compatibility and code sharing.
